pyproject.toml

```diff
@@ -36,6 +36,12 @@ dependencies = [
     "aiofiles",
     "aiohttp >= 3.0.0",
     "twiggy",
+    # We rely on deprecated features to maintain compat btw. pydantic v1 and v2
+    "pydantic < 3",
+    # pydantic already pulls it in, but we use it for TypedDict
+    "typing_extensions",
+    # Used for sending announcements
+    "pyperclip",
```

Comment:
This module doesn't seem to be maintained anymore (see for example https://github.com/asweigart/pyperclip/issues/248, the referenced issues, and the absence of any comments of the author in these issues). I don't like the idea of adding a dependency to something that's already dead.

Also it might make sense to make this dependency, or alternatives like [clipman](https://pypi.org/project/clipman/), optional, so that they only need to be installed if they're actually used.

Reply:
Ah, that's too bad about pyperclip. I'm still a bit partial to pyperclip, as I've used it a while, it's stable, and, unlike clipman, it has at least some basic unit tests. I can try out clipman and make it an optional dependency if you prefer, though.

Reply:
Meh, clipman fails with

```
Failed to setup the clipboard engine: Please install dbus-python package.
 - Via your system package manager. Possible package names: "python3-dbus-python" or "python3-dbus"
 - Or via PIP: "pip3 install dbus".
```

for me on KDE Plasma. It should fall back to xclip or wl-clipboard like pyperclip does, but instead it's failing, because it doesn't have the `dbus` module to communicate with klipper.

For now, I think I'll keep pyperclip and make it optional. We can always switch away from it or add a fallback if pyperclip starts causing problems.

Reply:
> For now, I think I'll keep pyperclip and make it optional. We can always switch away from it or add a fallback if pyperclip starts causing problems.

This was done in https://github.com/ansible-community/antsibull/pull/573/commits/28fb8ded5f4ff5d166cb935972e4b0fa3422538f. Let me know what you think of that solution.

Reply:
Looks great, thanks! It's annoying that the state of Python clipboard libraries seems to be somewhat a mess :)